cloudferrylib/base/action/action.py

```diff
@@ -37,3 +38,39 @@ def save(self):
 
     def restore(self):
         pass
+
+    def get_similar_tenants(self):
+        src_identity = self.src_cloud.resources[utils.IDENTITY_RESOURCE]
+        dst_identity = self.dst_cloud.resources[utils.IDENTITY_RESOURCE]
+
+        src_tenants = src_identity.get_tenants_list()
+        dst_tenants = dst_identity.get_tenants_list()
+
+        dst_tenant_map = {tenant.name.lower(): tenant.id for tenant in
+                          dst_tenants}
+
+        similar_tenants = {}
```

Comment:
This code will produce a lot of warnings in logs. Sergey, this code is alright, `find_by_name` don't fit here.